scripts/exgdas_global_marine_analysis_prep.py

```diff
@@ -149,8 +148,7 @@ def gen_bkg_list(window_begin=' ', bkg_path='.', file_type='gdas.t*.ocnf00[3-9]'
 # create output directory for soca DA
 anl_out = os.path.join(comout, 'ocnanal_'+os.getenv('CDATE'), 'Data')
 ufsda.mkdir(anl_out)
-ufsda.symlink(os.path.join(anl_dir, 'Data'), anl_out, remove=False)
-
+ufsda.symlink(anl_out, os.path.join(anl_dir, 'Data'), remove=False)
```

Comment:
I'm hoping to phase out `ufsda` in favor of `pygw`. I will not expect that of soca/atm just yet though but plan to try to do so for aero

Reply:
OK, thanks for the info.

Reply:
Resolving that one since you addressed that problem in one of the last few PR's @CoryMartin-NOAA .